apps/music/test/marionette/Player_test.js

```diff
@@ -133,4 +133,39 @@ marionette('Music player tests', function() {
       music.checkPlayerIconShown(true);
     });
   });
+
+  suite('Rating test', function() {
+    test('Rating persistance', function() {
```

Comment:
I based the testing scenario on the moztrap test. Maybe it is not a good test scenario.

It make sense. Do go further. Like Go back to the list, back to the player. Then restart the app back to the player. I'll do that.